features/registration.feature

```diff
@@ -20,3 +22,20 @@ Feature: Registration
   Scenario: passed token parameter without argument
     When I run `SUSEConnect --regcode --url http://localhost:3000`
     And the exit status should be 1
+
+  Scenario: passed invalid regcode
+    When I run `SUSEConnect --regcode foo --url http://localhost:3000`
```

Comment:
Why not `When I call SUSEConnect with '--regcode foo --url http://localhost:3000 arguments'`?
What's the difference?